hledger-web/hledger-web.cabal

```diff
@@ -183,6 +183,7 @@ library
     , text >=1.2
     , time >=1.5
     , transformers
+    , unix
```

Comment:
I haven't tested, but I think this will prevent finding an install plan on windows, and perhaps the solution is to replace that with `unix-compat` (and import from System.PosixCompat.Files).